front/components/sparkle/navigation.tsx

```diff
@@ -202,6 +204,15 @@ export const subNavigationBuild = ({
         subMenuLabel: current === "developers" ? subMenuLabel : undefined,
         subMenu: current === "developers" ? subMenu : undefined,
       },
+      {
```

Comment:
We want to show this only if you're gated into one Labs feature?

"Extract" as you probably noticed is also gated and marked as "Labs" for now. Ideally we rationalize how we show them (even if Extract is "only for us for now").

To gate a user in a feature you want to add a new FeatureFlag you want to add it to `WHITELISTABLE_FEATURES` in types/src/front/feature_flags 

`auth` will give you your flags if you ask kindly so you don't need any DB load to filter the display here :+1:

Reply:
I would prefix all solutions flags by `labs_solutions_...` that way we can use that convention here to show the labs menu entry

Reply:
<img width="314" alt="Screenshot 2024-04-10 at 17 39 17" src="https://github.com/dust-tt/dust/assets/1189312/306abb3e-1cc1-4060-b11a-e83f23f5b897">
@spolu something like that?

Reply:
More like moving the link to extract under the labs page and show the labs page only if relevant to the current user?

Reply:
Maybe BETA is more proper than ADVANCED?

Maybe call the section "Labs (beta)" and then "Call transcript" and "Extract"

Reply:
This works as well if easier and removes a page 👍

Reply:
Depends on how many labs features we're expecting to have in the mid-term

Reply:
We’re likely good with one menu entry per lab for now

Reply:
Ok, will do! In the future I'd advise against it though, discoverability isn't great if you have to click on all of them to know what they do - but since we're using feature flags, we can think that people will already know what it's doing for now.

Reply:
Moved extract under labs + added a feature flag for it (while keeping it available for dev or dust prod workspace - maybe we want to remove that part and only keep the feature flag for coherence?)